core/src/main/kotlin/dev/hotwire/core/turbo/http/HttpRepository.kt

```diff
@@ -0,0 +1,60 @@
+package dev.hotwire.core.turbo.http
+
+import android.webkit.CookieManager
+import dev.hotwire.core.logging.logError
+import dev.hotwire.core.turbo.util.dispatcherProvider
+import kotlinx.coroutines.withContext
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+
+internal class HttpRepository {
+    private val cookieManager = CookieManager.getInstance()
+
+    data class Result(
```

Comment:
Should the name be more specific like `HttpRequestResult` maybe? We already have a `Result` in `OfflineHttpRepository` plus they're both globally visible in the whole library namespace.

Reply:
I don't have too much concern with it, since they're nested within the corresponding `internal` classes, but I'll rename for clarity 👍